packages/services/api/src/modules/auth/resolvers/User.ts

```diff
@@ -0,0 +1,6 @@
+import type { UserResolvers } from './../../../__generated__/types.next';
+
+export const User: Pick<
+  UserResolvers,
+  'displayName' | 'email' | 'fullName' | 'id' | 'isAdmin' | 'provider'
+> = {};
```

Comment:
This is created automatically because we have a `UserMapper`. 
Currently, server preset still requires an empty object like this to do static analysis if there's a mapper - because if there's a mapper, there's a high chance user needs to write field mapping logic.